task-maker-lang/src/languages/cpp.rs

```diff
@@ -118,10 +118,6 @@ fn extract_includes(path: &Path) -> Vec<(PathBuf, PathBuf)> {
     lazy_static! {
         static ref RE: Regex = Regex::new(r#"#include\s*[<"]([^">]+)[>"]"#).expect("Invalid regex");
     }
-    let path = match path.canonicalize() {
-        Ok(path) => path,
-        _ => return vec![],
-    };
```

Comment:
I approve this changes, but do you mind writing a test for this? For example you can replicate the example I've shown. (I can do that too if you prefer.)

Reply:
Yeah I wrote a test for it, sorry I had some stuff happening and didn't have time.